rp_redcap/tasks.py

```diff
@@ -155,6 +156,7 @@ def run(self, project_id, **kwargs):
                         "name": contact.name or "",
                         "surname": contact.surname or "",
                         "title": contact.title or "",
+                        "week": int(utils.get_today().strftime("%W")),
```

Comment:
We could probably move `int(utils.get_today().strftime("%W"))` into `utils` as a `get_current_week_number` function.